l10n_it_withholding_tax_payment/i18n/l10n_it_withholding_tax_payment.pot

```diff
@@ -0,0 +1,268 @@
+msgid "Account"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.view_withholding_move_payment_form
+msgid "Account Setting"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__move_id
+msgid "Account move"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.wizard_wt_move_payment_create_view
+msgid "Cancel"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__company_id
+msgid "Company"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.view_withholding_move_payment_form
+msgid "Confirm"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: selection:withholding.tax.move.payment,state:0
+msgid "Confirmed"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.view_withholding_move_payment_form
+msgid "Create Account Move"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.actions.act_window,name:l10n_it_withholding_tax_payment.wizard_wt_move_payment_create_action
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.wizard_wt_move_payment_create_view
+msgid "Create Move Payment"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__create_uid
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_wizard_wt_move_payment_create__create_uid
+msgid "Created by"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__create_date
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_wizard_wt_move_payment_create__create_date
+msgid "Created on"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__date
+msgid "Date"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__date_payment
+msgid "Date Payment"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__date_start
+msgid "Date Start"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__date_stop
+msgid "Date Stop"
```

Comment:
"Start Date"
"Stop Date"

Reply:
👌

Reply:
Ehm... non ho capito perché hai tolto le maiuscole. :sweat_smile: 
In inglese ci vanno (in italiano no), sarebbe meglio mantenere lo standard Odoo.

Reply:
Beh... Ho tolto le maiuscole per un motivo preciso.

Il porting originario l'ho fatto senza ritoccare nulla (salvo dove necessario per il corretto funzionamento del modulo stesso): ho lasciato, di conseguenza, il resto esattamente così com'era.  
Se, però, mi viene richiesto di eseguire particolari modifiche, le andrò ad eseguire esattamente come le avrei fatte io.

Per cui, come puoi notare dai changes, ho sostituito, per esempio, alcuni apici singoli con i doppi (e viceversa) e ho utilizzato il case, secondo la mia opinione, in maniera corretta.

Ti chiedo, quindi...  
Qual è la regola corretta?

- Sempre e solo il case maiuscolo per tutte le parole indistintamente?
- Case maiuscolo solo le parole che non sono preposizioni?
- Mi baso sul case utilizzato originariamente per le stesse parole?
- Altro...?

Hai un link che mi descriva nel dettaglio questo standard di Odoo?

Grazie.

Reply:
> Hai un link che mi descriva nel dettaglio questo standard di Odoo?

L'uso delle maiuscole nei titoli (comprese le etichette) è una convenzione piuttosto diffusa della lingua inglese (in particolare americana).
Odoo si è attenuto a questa regola generale, infatti in tutti i moduli di Odoo CE i titoli e le etichette dei campi contengono parole con la prima lettera maiuscola.

In particolare questa cosa (per finalità di traduzione) è indicata nelle relative linee guida.

https://www.odoo-italia.org/documentazione/12.0/traduzioni/linee_guida/regole_grafiche.html#maiuscole

> Vuoi aprire tu, una PR alla nostra PR, in cui suggerisci direttamente le modifiche?

In realtà se riguardi tutti i suggerimenti che avevo indicato 4 giorni fa avevo già indicato le modifiche mantenendo la regola delle maiuscole, per questo avevo chiesto il motivo della modifica.

Reply:
> https://www.odoo-italia.org/documentazione/12.0/traduzioni/linee_guida/regole_grafiche.html#maiuscole

Buono a sapersi!  
Anche oggi ho imparato una cosa nuova! 🙃

---

Ecco fatto! Risolto! 👌